.github/workflows/package-release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           git add python_semantic_release_templates
           git config --global tag.gpgSign true
       - name: Python Semantic Release
-        uses: python-semantic-release/python-semantic-release@v8.2.0  # 8.3.0 has a bug in the version output variable
+        uses: python-semantic-release/python-semantic-release@v8.5.0  # 8.3.0 has a bug in the version output variable
```

Comment:
This needs to be tested first.

Reply:
I tested this on my fork, it no longer has the bug in the version output variable.